### ISSUE TITLE
Support for nx 14

### ISSUE THREAD

**Issue by SnakeSVx** [twittwer/nx-tools#64]
We are currently in the proces of upgrading from nx 13 to nx 14 and we are encountering some issues with npm and the double versions of nx that seem to be available in node_modules.

Because of the dependency of these tools on nx 13, npm is installing both the v13 as the v14 bin and depending on the environment it tries to use wrong 'link' to make it accesible in node_modules/bin/...

Would it be possible to update this tool with support for v14 to prevent this version conflict?

```
npm ERR! code 1
npm ERR! path project_path/node_modules/@twittwer/compodoc/node_modules/nx
npm ERR! command failed
npm ERR! command sh -c node ./bin/init
npm ERR! >  NX   Cannot read properties of undefined (reading 'endsWith')
npm ERR! A complete log of this run can be found in:
```

**Comment by SnakeSVx** [comment]
Currently we've been able to bypass this by forcing a v14 install using the following in package.json

```
  "overrides": {
    "@nrwl/devkit": "14.1.5"
  }
```

**Comment by twittwer** [comment]
Thx to @arturovt for providing an PR with the NX 14 update.
It's published in version 1.8.0 with all dependencies updated.
